application/server/notification/scan_notifier.go

```diff
@@ -117,6 +117,34 @@ func (n *scanNotifier) appendOssIssues(scanIssues []lsp.ScanIssue, folderPath st
 			continue // skip non-oss issues
 		}
 
+		matchingIssues := make([]lsp.OssIssueData, len(additionalData.MatchingIssues))
+		for i, matchingIssue := range additionalData.MatchingIssues {
```

Comment:
This now includes the matching issues in the type, so that both the HTML renderer using in. VSCode and the native UI in the IntelliJ plugin can use it to build the following:
![Screenshot 2024-05-09 at 13 38 39](https://github.com/snyk/snyk-ls/assets/81559517/3d907175-47c5-4bf9-8ba6-99cf91c7d5ed)